doc/development.md

```diff
@@ -117,6 +116,11 @@ Before we can start you need to match some prerequirements on your host machine.
        done
        ```
 
-    4. Run the Rails server ```bin/rails s -b 0.0.0.0 -p 5000```
+    4. shutdown the production server `systemctl stop crowbar && systemctl disable crowbar`
```

Comment:
> What kind of confusion?

production and development (2 different apps) use one database. can you imagine the confusion when you use both?